config/about.config.ts

```diff
@@ -27,16 +27,16 @@ export const aboutPageData: AboutPageData = {
     projects: {
       section: 'projects',
       description:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Leo vel fringilla est ullamcorper. Sagittis id consectetur purus ut faucibus pulvinar elementum.',
-      links: [{ name: 'apply today', link: '/about' }],
+        'ACM Projects is a semester-long guided projects initiative for those new to software development, teaching collaborative tools, new technologies, and core development principles complimentary to curriculum.',
+      links: [{ name: 'apply today', link: '/projects' }],
```

Comment:
Should the name be something else other than `apply today`? Doesn't seem to make sense to name the button that way when it does not redirect users to application form.